intera_interface/src/intera_motion_interface/motion_trajectory.py

```diff
@@ -70,14 +70,22 @@ def __init__(self, label = None,
     def stop_trajectory(self):
         self._client.stop_trajectory()
 
-    def send_trajectory(self):
+    def send_trajectory(self, wait_for_result=True):
```

Comment:
This now needs a `@param` parameter description and an `@return` return description